src/lib/tvgLoadModule.h

```diff
@@ -49,7 +50,7 @@ class LoadModule
 
     virtual bool read() = 0;
     virtual bool close() = 0;
-    virtual unique_ptr<Surface> bitmap() { return nullptr; }
+    virtual unique_ptr<Surface> bitmap(bool changeColorSpace) { return nullptr; }
```

Comment:
Thinking about this... 
changeColorSpace is too ambiguous. In the end, the requested colorspace value has to be specified. 

How about this?
```
enum Colorspace { RGBA, ... };
class Coverter { 
  static RGBA2ABGR();
  ...
 };  //in tvgRender.h
...
LoadModule::bitmap(Colorspace cs);
```

Reply:
I changed the bool changeColorSpace to the colorSpace value and compare it inside the loader.
The colorspace change used now is ARGB <-> ABGR.
ARGBtoABGR or ABGRtoARGB have the same logic, so I think, The current logic doesn't have enough functionality to make it a class.
Some of the common functions can be moved to the header, but I think it is in the same context as _premultiply(), and after this PR, we can work on a separate PR to organize these.

Reply:
Ok, actually both are spagetties. Back then I allowed it because I have no better idea at that time. Even now. (Do you?) However colorspace converter is different. changeColorSpace() with out any specification is too ambugiuous because there will be more colorspace formats such as RGB, ARGB, BGR, BGRA, GRAYSCALE, etc. 

It's my opinion. Overall, I agree on your contibution code, anyway it helps ThorVG better than before. But I would like to let you know there is a possibibility to revise the whole base code.